src/qd8-f32-qb4w-gemm/gen/qd8-f32-qb4w-gemm-1x8c2-minmax-avx2.c

```diff
@@ -0,0 +1,127 @@
+// Auto-generated file. Do not edit!
```

Comment:
why do we need c2? is it faster?

Reply:
Commented below with more context. I removed it, since I didn't intend to include it.